src/FetchRequestPerformer.js

```diff
@@ -0,0 +1,11 @@
+/* @flow */
+
+import type { RequestPerformer } from './types';
+
+require('fetch-everywhere');
+
+class FetchRequestPerformer implements RequestPerformer<Request, Response> {
```

Comment:
I forgot what `RequestPerformer<Request, Response>` mean in Flow. 

@webfella and ivan should look this up.

Reply:
Okay so this is the parameterized generic syntax. I guess my question is that I am not sure how flow will uses these generics in the call site. Thoughts @eliperkins

Reply:
`<Request, Response>` are the specialized generics that `RequestPerformer` expects. Specifically, `Request` is the input to this class and `Response` is the output.

Reply:
Actually I think I kind of get from looking at TokenRequestFactory, but I don't think I understand it will enough to explain it.

Reply:
@iwillig Client takes in three dependencies when constructing it:

- `RequestFactory`
- `RequestPerformer`
- `ResponseParser`

These classes map to a flow somewhat like:

`UserlandCode ---(RequestFactory)---> Request ---(RequestPerformer)---> Response ---(ResponseParser)---> ClubhouseType`

The classes defined here are the glue between those transitions.